codegen/codegen.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	defaultURL = "https://releases.rancher.com/kontainer-driver-metadata/dev-v2.9/data.json"
+	defaultURL = "https://raw.githubusercontent.com/vardhaman22/rancher-kontainer-driver-metadata/k8s-1.30/data/data.json"
```

Comment:
if we can get https://github.com/rancher/kontainer-driver-metadata/pull/1406 to be merged first, then do not need to change the URL here, right?

Reply:
yeah, once kdm is merged, this url change would be removed. I asked for using the custom branch url to verify build is passing beforehand